nilearn/_utils/extmath.py

```diff
@@ -75,3 +79,61 @@ def is_spd(M, decimal=15, verbose=1):
     if not ispd and verbose > 0:
         print("matrix has a negative eigenvalue: %.3f" % eigvalsh.min())
     return ispd
+
+
+def check_threshold(threshold, data, percentile_calculate, name='threshold'):
```

Comment:
Agreed with @lesteve : I think that it should be a private function, and be put close to where it is currently used.